frontend/dashboard/pages/CreateService/CreateService.test.tsx

```diff
@@ -63,18 +61,18 @@ describe('CreateService', () => {
     await waitFor(() => {
       expect(screen.getByRole('combobox')).toBeInTheDocument();
       expect(screen.getByRole('combobox')).toBeDisabled();
-      expect(screen.getByRole('combobox')).toHaveTextContent('user_full_name');
     });
   });
 
   it('should show error message that app name is too long when it exceeds max length', async () => {
     const user = userEvent.setup();
-    render();
+    const { container } = render();
 
     await waitForElementToBeRemoved(() => screen.getByText('dashboard.loading'));
 
+    await act(() => user.click(screen.getByRole('combobox')));
     await act(() => user.click(screen.getByRole('option', { name: /user_full_name/i })));
-    await act(() => user.type(screen.getByRole('textbox'), 'this-app-name-is-longer-than-max'));
+    await act(() => user.type(container.querySelector('#service-saved-name'), 'this-app-name-is-longer-than-max'));
```

Comment:
Hva er grunnen til at vi ikke kan bruke screen og en selector som også sjekker på a11y? :)

Reply:
Det er to tekstbokser, så testen må finne den riktige.